### DISCUSSION TITLE
What do the 📈 and bookmark icons mean?

### DISCUSSION THREAD

**Discussion by twoloavesofbread** [CappielloAntonio/tempo#186] [Q&A]
On the media player, when you touch the album art, a few options come up. I can glean the meanings of the top icons -- the first is clearly download, second is add to playlist. I'm curious what the two below icons mean:

<img src="https://github.com/CappielloAntonio/tempo/assets/46307093/4300c029-b882-4f04-bb62-f045db9916f4" width="300">

Tapping the first one appears to do nothing. Tapping the second one says "Salvato" despite being in EN-US. I presume it's being saved, but I'm unsure how.

**Comment by ZenKyma** [comment]
I also wonder...

**Comment by foliva** [comment]
Same here ... Though one of the options is different for me.

![Screenshot_20241205-170912~2](https://github.com/user-attachments/assets/134c4a57-c821-4736-bb1b-d51ec2f6e0e7)

**Comment by candideu** [comment]
Has anyone figured this out? I'm so curious lol

**Comment by eddyizm** [comment]
<img width="159" height="176" alt="image" src="https://github.com/user-attachments/assets/824fa12e-9f48-4d93-a2be-df883e5d94bb" />

so I marked the icons with numbers for clarity. Also note, these specific actions are on my fork and a lot of this stuff was partially broken (bugs) or not working as expected.

1. Downloads the track (there is a notification if the android screen but not a pop toast currently )
2. Adds track to playlist - pops up playlist dialog.
3. Adds tracks to the queue via instant mix function
4. Saves play queue (if the feature is enabled in the settings) 
4.5 if the setting is not enabled which @foliva above showed his screen, it toggles a view of the lyrics if available (slides to the right) 

I have fixed a few bugs but this needs a lot more work. I will add this to the [documentation](https://eddyizm.github.io/tempo/USAGE.html) which I need help with and in case any of you are interested, I have updated releases over there -> https://github.com/eddyizm/tempo/releases/tag/v3.16.6
